jmix-security/security-flowui/src/main/resources/io/jmix/securityflowui/view/resetpassword/reset-password-view.xml

```diff
@@ -15,22 +15,37 @@
   ~ limitations under the License.
   -->
 
-<view xmlns="http://jmix.io/schema/flowui/view"
-      title="msg://resetPasswordView.resetPasswordsTitle">
+<view xmlns="http://jmix.io/schema/flowui/view">
+    <data>
+        <collection id="userPasswordValueDc"
+                    class="io.jmix.securityflowui.view.resetpassword.model.UserPasswordValue">
+            <loader id="userPasswordValueDl"/>
+        </collection>
+    </data>
     <actions>
-        <action id="saveAction"
-                actionVariant="PRIMARY"
-                icon="CHECK"
-                text="msg://resetPasswordView.generateSave"/>
-        <action id="closeAction"
-                icon="CLOSE"
-                text="msg:///actions.Close"/>
+        <action id="closeAction" type="view_close"/>
     </actions>
     <layout>
-        <p id="resetPasswordTooltip" text="msg://resetPasswordView.printPasswords"/>
-        <textArea id="readyPassword" visible="false" width="100%"/>
+        <vbox padding="false" spacing="false">
+            <span id="resetPasswordLabel" text="msg://resetPasswordView.resetPasswordLabel.text"/>
+            <span id="resetPasswordSubLabel" text="msg://resetPasswordView.resetPasswordSubLabel.text"
+                  classNames="text-secondary, text-s"/>
+        </vbox>
+        <dataGrid id="passwordsDataGrid"
+                  dataContainer="userPasswordValueDc"
+                  height="100%"
```

Comment:
remove `height="100%"` and `minHeight`. Replace them with:

```
allRowsVisible="true"
maxHeight="15em"
```

This will help you to get rid of empty space

<img width="595" alt="Screenshot 2024-09-10 at 13 44 37" src="https://github.com/user-attachments/assets/65a811f3-0171-4fb6-a2e6-ab187155af16">